RRemoteConfig/Poller.swift

```diff
@@ -0,0 +1,38 @@
+internal class Poller {
+    let delay: TimeInterval
+    var timer: Timer?
+
+    init(delay: TimeInterval = 60.0 * 60.0) {
+        self.delay = delay
+    }
+
+    func start(action: @escaping () -> Void) {
+        // initial fetch
+        action()
+
+        self.timer = Timer(timeInterval: self.delay, repeats: true, block: { (_) in
+            action()
+        })
+
+        // polling
+        if let timer = self.timer {
+            timer.tolerance = 0.1 * self.delay
+            DispatchQueue.main.async {
+                RunLoop.current.add(timer, forMode: .common)
+            }
+        }
+    }
+
+    func stop() {
```

Comment:
What case will the `stop` function be used for?

Reply:
Currently unused but I wanted to be explicit that the timer needs to be invalidated on the same thread it had been added to the runloop. I can remove it if you prefer and just make that a comment when it is added to the runloop?

Reply:
Removed `stop` and added comment instead.